### ISSUE TITLE
Night elf character creation error

### ISSUE THREAD

**Issue by gabbyelf99** [dalaranwow/dalaran-wow#5544]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Character creation, new account, can't create Night elf character, Error creating character
**Expected behaviour**: Tell us what should happen instead
Should create Night elf character
**Steps to reproduce the problem**:

1. Try to create new Night elf character
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Golganeth** [comment]
We do not recommend using a custom or modified WoTLK client, they tend to cause errors very often.

Visit this [>link<](http://www.dalaran-wow.com/forums/input-and-assistance/community-support/topic/11769/help-can-t-create-a-new-character) for further information. Quoting:

>Hail!
>
>Custom clients (such as those that have WoD models) are known to cause client instability and some errors when you try to create a character. Please use a normal WotLK client to avoid this kind of problems.
>
>You can download and install our client, which can be found [here](http://www.dalaran-wow.com/game-client)
>
>Cheers!

If you do not wish to install our normal client, you could try to create the character using a different face, hair style, etc., although be warned from the possible errors you may be encountering in the future. 
Also, keep in mind the visual modifications from the client are only visible to you, and not the rest of the players.

Closing.
